librz/debug/p/debug_bf.c

```diff
@@ -120,9 +120,9 @@ static int rz_debug_bf_continue_syscall(RzDebug *dbg, int pid, int num) {
 	return true;
 }
 
-static int rz_debug_bf_wait(RzDebug *dbg, int pid) {
+static RzDebugReasonType rz_debug_bf_wait(RzDebug *dbg, int pid) {
 	/* do nothing */
-	return true;
+	return RZ_DEBUG_REASON_SIGNAL;
```

Comment:
This should be REASON_NONE to avoid the "child received signal" log. You can even remove the entire function since it doesn't do anything and `rz_debug_wait` will return REASON_UNKNOWN.

Reply:
Same issue in debug_esil/io/rap